tests/modules/layers/test_normalizations.py

```diff
@@ -20,3 +28,45 @@ def test_fp32groupnorm():
     norm = Fp32GroupNorm(2, 4)
     output = norm(x)
     assert output.dtype == torch.float16
+
+
+def test_rms_norm_fp32return():
```

Comment:
why dont we just do 1 test with fp32 and assert on output values?

Reply:
my idea was to separate out the tests esp if we add more dtype support (ala fp8) but will merge this into one for brevity.